src/Native/LIBMFNative/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+project (LIBMFNative)
+
+include_directories(libmf)
+
+set(SOURCES
+    UnmanagedMemory.cpp
+    libmf/mf.cpp
+)
+
+if(WIN32)
```

Comment:
This can be simplified to `if(NOT WIN32)`

Reply:
Fixed. Thanks.

---
In reply to: [226720783](https://github.com/dotnet/machinelearning/pull/1263#discussion_r226720783) [](ancestors = 226720783)